src/generator/AutoRest.AzureResourceSchema/AzureResourceSchemaCodeGenerator.cs

```diff
@@ -50,7 +50,13 @@ public override void NormalizeClientModel(ServiceClient serviceClient)
                 ResourceSchemaWriter.Write(stringWriter, resourceSchemas[resourceProvider]);
 
                 string schemaPath = Path.Combine(Settings.OutputDirectory, resourceProvider + ".json");
-                await Write(stringWriter.ToString(), schemaPath);
+                await Write(stringWriter.ToString(), schemaPath, true);
+
+                stringWriter = new StringWriter();
+                ResourceMarkdownWriter.Write(stringWriter, resourceSchemas[resourceProvider]);
```

Comment:
Why did you choose to make this part of the AzureResourceSchema generator, and not its own generator?